WireNetwork/Tests/WireNetworkTests/APIs/MLSAPI/MLSAPITests.swift

```diff
@@ -180,6 +180,21 @@ final class MLSAPITests: XCTestCase {
             try await api.postCommitBundle(Scaffolding.commitBundle)
         }
     }
+
+    // MARK: - Reset MLS conversation
+
+    func testResetMLSConversation_SuccessResponse_V9() async throws {
```

Comment:
todo: add error case for the mlsReset conversation 400

status code: 400 label: mls-invalid-leaf-node-index

status code: 400 label: mls-invalid-leaf-node-signature